experimental/packages/otlp-transformer/package.json

```diff
@@ -6,10 +6,48 @@
   },
   "version": "0.56.0",
   "description": "Transform OpenTelemetry SDK data into OTLP",
-  "module": "build/esm/index.js",
-  "esnext": "build/esnext/index.js",
-  "types": "build/src/index.d.ts",
-  "main": "build/src/index.js",
+  "exports": {
```

Comment:
Another alternative to this PR could be to modify this to expose the generated code: https://github.com/open-telemetry/opentelemetry-js/pull/5259/files#r1884673988